templates/test/ci/prow-dual-stack/kustomization.yaml

```diff
@@ -3,6 +3,7 @@ kind: Kustomization
 namespace: default
 resources:
   - ../../../flavors/dual-stack
+  - machine-pool-dualstack.yaml
```

Comment:
Wondering if we could maintain `machine-pool-dualstack.yaml` in `/templates/flavors/dual-stack/` so there will be one less file to maintain? Or is this file exclusive for prow ?

Reply:
the machine pool dual stack combination is only used in prow for testing so we get test coverage for this specific combination of features (the dual stack flavor just has machinedeployments, since machinepools are experimental and behind a feature flag)